src/components/image/index.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Image from ".";
+
+<Meta title="Components/Image" component={Image} />
+
+# Image
+
+Image provide a vital part of story telling for websites. Images are used to complement text, or provide supporting information.
+Images can be used as informational or for decoration. This component will provide the means to display images in a variety of different ways.
+
+## Usage
+
+```
+import { Image } from '@wpmedia/arc-themes-components';
+
+
+<Image src="cat.jpg" alt="A cat jumping up to a ball" />
+```
+
+## Properties
+
+<ArgsTable of={Image} />
+
+## Stories
+
+** Default **
+
+<Canvas>
+	<Story name="Default">
+		<Image src="/computer-user.jpeg" alt="A person sat down using a laptop computer" />
+	</Story>
+</Canvas>
+
+** Image not found **
+
+<Canvas>
+	<Story name="Image not found">
+		<Image src="cat.jpg" alt="A cat jumping up to a ball" />
+	</Story>
+</Canvas>
+
+** Width and Height **
+
+<Canvas>
+	<Story name="Width and Height">
+		<Image
+			src="/computer-user.jpeg"
+			alt="A person sat down using a laptop computer"
+			width="200"
+			height="147"
+		/>
+	</Story>
+</Canvas>
```

Comment:
Do we want some iterations like width/height only?

```
** Width only **

<Canvas>
	<Story name="Width only">
		<Image
			src="/computer-user.jpeg"
			alt="A person sat down using a laptop computer"
			width="250"
		/>
	</Story>
</Canvas>

** Height only **

<Canvas>
	<Story name="Height only">
		<Image
			src="/computer-user.jpeg"
			alt="A person sat down using a laptop computer"
			height="250"
		/>
	</Story>
</Canvas>
```

Reply:
Added @nschubach